themes/default/content/docs/get-started/aws/next-steps.md

```diff
@@ -14,37 +14,34 @@ menu:
 aliases: ["/docs/quickstart/aws/next-steps/"]
 ---
 
-You've seen how to quickly get started using AWS with Pulumi.
+Congrats! You have deployed your first project on AWS with Pulumi. Here are the next steps:
 
-From here, you can dive deeper with additional AWS tutorials:
+## Learn Pulumi
+You can go through our Learn tutorials to have a comprehensive walkthrough of the key Pulumi concepts taught through the context of a real-life application.
 
-* [Containers on ECS "Fargate"]({{< relref "/registry/packages/aws/how-to-guides/ecs-fargate" >}}): Deploy containers to Amazon
-* [EC2 Linux WebServer VM]({{< relref "/registry/packages/aws/how-to-guides/ec2-webserver" >}}): Create an EC2 Linux Web Server virtual machine
-* [Serverless REST API Gateways using Lambda]({{< relref "/registry/packages/aws/how-to-guides/rest-api" >}}): Create simple RESTful web server using AWS Lambdas
-* [Serve a Static Website from S3]({{< relref "/registry/packages/aws/how-to-guides/s3-website" >}}): Serve a static website out of content in an S3 bucket
-* [Serverless + Containers + Infrastructure]({{< relref "/registry/packages/aws/how-to-guides/video-thumbnailer" >}}): Deploy a complete  application using a combination of buckets, serverless functions and containers.
+<div class="flex justify-left py-6">
+    <a class="btn btn-lg mx-1 my-1" href="{{< relref "/learn/pulumi-fundamentals/" >}}">Learn Pulumi Fundamentals</a>
```

Comment:
It's better to only have one button of a primary color per page (one primary CTA) to help the user with other buttons being in secondary colors/secondary button types. Did the layout go through UX?